app/assets/stylesheets/modules/categories.scss

```diff
@@ -1,86 +1,23 @@
 .categories {
-  height: auto;
-  margin-left: 5%;
-  position: absolute;
-  top: 75px;
+  float: left;
+  margin: 50px 0 0 5%;
   width: 10%;
-  &.index { top: 22%; }
 
-  @include mq-tablet() {
-    margin-top: -90px;
-    position: static;
-  }
-
-  a { text-decoration: none; }
-  .title {
-    color: $gray-medium-lighter;
-    font-size: 16px;
-    font-family: 'Roboto Light';
-  }
-  .category {
-    color: $gray-medium-light;
-    font-size: 20px;
+  h1, h2, a {
```

Comment:
I would prefer you to create a new class and assign it to the elements you want instead of using the elements; just think about it and if it is feasible ... go ahead and do it